doc/whats_new/v0.20.rst

```diff
@@ -142,6 +142,10 @@ Classifiers and regressors
   several times in a row. :issue:`9234` by :user:`andrewww <andrewww>`
   and :user:`Minghui Liu <minghui-liu>`.
 
+- In both :class:`ensemble.RandomForestClassifier` and `ensemble.ExtraTreesClassifier`,
```

Comment:
This should be under API changes, rather than enhancements.

Reply:
Fixed this too in my latest commit.